xml/cha_images.xml

```diff
@@ -273,9 +289,12 @@
  <sect1>
   <title>&pint;</title>
   <para>
-   The &pint; (&pinta;) provides information about the images and servers &suse;
-   publishes in supported public cloud frameworks. &pinta; is available at
-   <link xlink:href="&pinturl;"/>.
+   The &pint; (&pinta;) provides information about the images &suse;
+   publishes and servers that are part of the &suse; operated update
+   infrastructure. &pinta; is available at
+   <link xlink:href="&pinturl;"/> and provided as an API and commandline
+   tool with the <package>python3-susepubliccloudinfo</package> package
+   from the <repository>Public Cloud Module</repository> repository.
```

Comment:
This does not validate, there is no `repository` tag in DocBook. Use `systemitem` or simply `literal` instead.
```suggestion
   from the <systemitem>Public Cloud Module</systemitem> repository.
```